gamedefs.h

```diff
@@ -771,6 +777,18 @@ class GameDefs {
 
 	// How to count faction activit. See FactionActivty enum for more details.
 	FactionActivityRules FACTION_ACTIVITY;
+
+	// how buildings are destroyed, read more in DestroyBehavior enum
+	DestroyBehavior DESTROY_BEHAVIOR;
+
+	// set folowing value when DESTROY_BEHAVIOR is not INSTANT
+	// how much structure points can be destroyed per turn
+	// MAX_DESTROY_PERCENT accepts values from [1..100]
+	// MIN_DESTROY_POINTS accepts [0..infinity]
+	// formula:
+	//     turn destroyable points = max(MIN_DESTROY_POINTS, Max Structure Points * MAX_DESTROY_PERCENT / 100)
+	int MIN_DESTROY_POINTS;
```

Comment:
can you please explain about min destroy points and why they needed?

Reply:
If destruction is not instant then it helps to control how fast buildings can be destroyed. 

You can make so that small buildings can be destroyed in a turn, but huge one's will need several turns no matter what.

Current example with 34% and min 200 point destroyble points will make citadel required 3 turns to fully destroyed,  castle 2 turns, and the rest of the structures will need 1 turn. if sufficient man count of course.

Reply:
Setting % to 100 will make all buildings destroyeable in a turn